src/pages/docs/support/log-files.md

```diff
@@ -47,7 +47,7 @@ The verbosity of file logging is controlled in the `octopus-log-file` section:
 
 The `minlevel` attribute is most useful for configuring the logging level. Change this value to `Trace` and set `OCTOPUS__Logging__File__LogEventLevel` environment variable to `Verbose` to gather more information.
 
-The Octopus process will automatically switch to the new logging level as soon as the file is saved.
+This process requires a server restart in order to apply the changes to the environment variables.
```

Comment:
The original statement,
> The Octopus process will automatically switch to the new logging level as soon as the file is saved.

, is referring to the changes for `octopus.server.exe.nlog`, which is accurate. ([ref](https://octopus.com/docs/support/log-files))

Reply:
The change to .nlog file is applied as soon as the file is saved. However, the config alone in nlog would not help if `OCTOPUS__Logging__File__LogEventLevel` and the new environment variable are not set.

Reply:
Leaving that statement will cause confusion to users as they will think the changes to the environment variables are also automatically applied.

Reply:
Discussion: https://octopusdeploy.slack.com/archives/C056PF0S876/p1690179264126229